app/helpers/audits_helper.rb

```diff
@@ -43,6 +43,8 @@ def id_to_label(name, change, audit: @audit, truncate: true)
   def audit_title(audit)
     type_name = audited_type audit
     case type_name
+      when 'Host Ansible Role'
+        audit.associated_name + " / " + id_to_label("ansible_role_id", audit.audited_changes["ansible_role_id"], audit: audit)
```

Comment:
This is not exactly fortunate considering ansible roles are something that lives in a plugin. Let's talk about it next time we see each other